blivet/devices/lvm.py

```diff
@@ -802,7 +852,7 @@ def metadata_vg_space_used(self):
         if non_raid_base and self.is_raid_lv:
             zero_superblock = lambda x: Size(0)
             try:
-                raided_space = self._raid_level.get_space(non_raid_base, self._num_raid_pvs,
+                raided_space = self.raid_level.get_space(non_raid_base, self.num_raid_pvs,
                                                           superblock_size_func=zero_superblock)
```

Comment:
`E127 continuation line over-indented for visual indent`

This is last pylint/pep warning. There are some more, but these exist on 3.0-devel too.